javascript-create-module/template/settings/definitions.cnd

```diff
@@ -1,17 +1,12 @@
 <jnt = 'http://www.jahia.org/jahia/nt/1.0'>
 <jmix = 'http://www.jahia.org/jahia/mix/1.0'>
-<$$MODULE_NAMESPACE$$ = 'http://www.acme.org/$$MODULE_NAME$$/nt/1.0'>
-<$$MODULE_NAMESPACE$$mix = 'http://www.acme.org/$$MODULE_NAME$$/mix/1.0'>
+<$MODULE = 'https://example.com/$MODULE/1.0'>
```

Comment:
what if the MODULE contains a space? Would that work? 🤔

Reply:
It's camelCased! This case is properly tested (I updated the tests you wrote)